libkbfs/tlf_journal.go

```diff
@@ -990,19 +1015,24 @@ func (j *tlfJournal) getJournalStatusWithPaths(ctx context.Context,
 			// but don't cache it.
 			unflushedPaths = make(unflushedPathsMap)
 			j.log.CDebugf(ctx, "Making incomplete unflushed path cache")
-			irmds, err := j.batchConvertImmutables(ctx, ibrmds)
+			mdInfos, err := j.getUnflushedPathMDInfos(ctx, ibrmds)
 			if err != nil {
 				return TLFJournalStatus{}, err
 			}
 			err = addUnflushedPaths(ctx, j.uid, j.key.KID(),
-				j.config.Codec(), j.log, irmds, cpp,
+				j.config.Codec(), j.log, mdInfos, cpp,
 				unflushedPaths)
 			if err != nil {
 				return TLFJournalStatus{}, err
 			}
 			break
 		}
 
+		if len(ibrmds) == 0 {
```

Comment:
I had thought there was, in `addUnflushedPaths`, but I didn't see that only do `mdInfos[len(mdInfos)-1]` after the `processedOne` check. Removed!